eng/pipelines/build-windows-on-unix-job.yml

```diff
@@ -0,0 +1,51 @@
+# Build on Unix
+parameters:
+- name: jobName
+  type: string
+  default: ''
+- name: testArtifactName
+  type: string
+  default: ''
+- name: configuration
+  type: string
+  default: 'Debug'
+- name: queueName
+  type: string
+  default: ''
+- name: vmImageName
+  type: string
+  default: ''
+
+jobs:
+- job: ${{ parameters.jobName }}
+  pool:
+    ${{ if ne(parameters.queueName, '') }}:
+      name: NetCore1ESPool-Public
+      demands: ImageOverride -equals ${{ parameters.queueName }}
+
+    ${{ if ne(parameters.vmImageName, '') }}:
+      vmImage: ${{ parameters.vmImageName }}
+  timeoutInMinutes: 40
+
+  steps:
+    - template: checkout-unix-task.yml
+
+    - script: ./eng/build.sh --ci --restore --prepareMachine --binaryLog --configuration ${{ parameters.configuration }} --build-full-solution
+      displayName: Restore
+
+    - script: ./eng/build.sh --ci --build --publish --pack --prepareMachine --binaryLog --skipDocumentation --configuration ${{ parameters.configuration }} --build-full-solution
```

Comment:
I was wondering if the artifacts we build for windows are just a superset in practice, or if there's something fundamentally different about them? Basically, could we eliminate the Build Unix Debug job to reduce pressure on the build queue.

Reply:
Makes sense. Let's file it away for later.